src/theme/components/Terminal/rpc/GetBlock.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
+
+export default [
+    <span data-ty="input">{`curl -d "{\"sequence\": 1}" -X POST http://localhost:8080/chain/getBlock`}</span>,
```

Comment:
it looks like the " are not escaped here in the final output on the website. we may need to double escape 😆

Reply:
I didn't see your comment about this above, resolving this